### PR TITLE
Feature/analytics persistance entities

### PR DESCRIPTION
**Pull request by brettminnie**

### CODE REVIEW

File: src/TransformCore/PHE/AnalyticsBundle/Entity/AnalyticsExport.php [hunk 1]
```diff
@@ -0,0 +1,430 @@
...
+        return $this;
+    }
+
+    /**
+     * @return float
+     */
+    public function getPercentageCompletion()
+    {
+        $value = (empty($this->getStarts()))
+            ? 0 : ($this->getCompletions() / $this->getStarts()) * 100;
```
Comment (eddiejaoude) [inline src/TransformCore/PHE/AnalyticsBundle/Entity/AnalyticsExport.php]: float? Percent is 0 - 100, integer would be better?

Comment (brettminnie) [inline src/TransformCore/PHE/AnalyticsBundle/Entity/AnalyticsExport.php]: Nope, a percentage is a float

Comment (eddiejaoude) [inline src/TransformCore/PHE/AnalyticsBundle/Entity/AnalyticsExport.php]: True, if you want decimal :smile:

Comment (brettminnie) [inline src/TransformCore/PHE/AnalyticsBundle/Entity/AnalyticsExport.php]: The end user can round it to an int if they want :)

**Pull request merged**